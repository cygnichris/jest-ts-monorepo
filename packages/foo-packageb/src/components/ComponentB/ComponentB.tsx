import * as React from 'react';

export const ComponentB = () => <div>Hello World </div>
