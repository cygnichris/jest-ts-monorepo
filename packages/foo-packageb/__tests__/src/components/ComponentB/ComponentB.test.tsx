import * as React from 'react';
import { render } from '@testing-library/react';
import { ComponentB } from '@bsrc/components/ComponentB/ComponentB';

describe('<ComponentB/> Snapshot', () => {
  it('renders', () => {
    const { asFragment } = render(
      <ComponentB />
    );

    expect(asFragment()).toMatchSnapshot();
  })
})
