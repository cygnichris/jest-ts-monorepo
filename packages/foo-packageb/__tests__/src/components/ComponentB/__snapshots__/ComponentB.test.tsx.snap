// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ComponentB/> Snapshot renders 1`] = `
<DocumentFragment>
  <div>
    Hello World 
  </div>
</DocumentFragment>
`;
